README.md

```diff
@@ -153,6 +156,10 @@ it simple and easy. So, it has some limitations:
 - No advanced feature, like P2P replication between several cozy-desktop
   instances.
 
+## License
+
+Cozy Desktop Client is developed by Cozy Cloud and distributed under the AGPL v3 license.
```

Comment:
@frankrousseau under which license is cozy-desktop? Simon has put AGPL v3 in the README and LGPL in package.json. And I don't remember which one is the good one.